### DISCUSSION TITLE
How this repo merge v5.0.0-alpha version changelog?

### DISCUSSION THREAD

**Discussion by whatwg6** [lerna/lerna#3334] [Q&A] [answered]
### In [v5.0.0-alpha.2 changelog ](https://github.com/lerna/lerna/blob/v5.0.0-alpha.2/CHANGELOG.md) has this changelog below

[5.0.0-alpha.2](https://github.com/lerna/lerna/compare/v5.0.0-alpha.1...v5.0.0-alpha.2) (2022-05-24)
Note: Version bump only for package lerna-monorepo

[5.0.0-alpha.1](https://github.com/lerna/lerna/compare/v5.0.0-alpha.0...v5.0.0-alpha.1) (2022-05-24)
Note: Version bump only for package lerna-monorepo

[5.0.0-alpha.0](https://github.com/lerna/lerna/compare/v4.0.0...v5.0.0-alpha.0) (2022-05-23)
Bug Fixes
replace read-package-tree with @npmcli/arborist (https://github.com/lerna/lerna/issues/3133) ([f1c10a4](https://github.com/lerna/lerna/commit/f1c10a40fe667e5012a60e9b372a8e7996465de1))
resolve most dependency audit issues (https://github.com/lerna/lerna/issues/3127) ([2b3b191](https://github.com/lerna/lerna/commit/2b3b1910d377122f294b4a79a291596fbc81241c))

### In [v5.0.0 changelog](https://github.com/lerna/lerna/blob/v5.0.0/CHANGELOG.md) version 5.0.0-alpha.0 ~ 5.0.0-alpha.2 changelog was merge to v5.0.0

- [5.0.0](https://github.com/lerna/lerna/compare/v4.0.0...v5.0.0) (2022-05-24)
Bug Fixes
replace read-package-tree with @npmcli/arborist (https://github.com/lerna/lerna/issues/3133) ([f1c10a4](https://github.com/lerna/lerna/commit/f1c10a40fe667e5012a60e9b372a8e7996465de1))
resolve most dependency audit issues (https://github.com/lerna/lerna/issues/3127) ([2b3b191](https://github.com/lerna/lerna/commit/2b3b1910d377122f294b4a79a291596fbc81241c))

### I have the same need, how do I do it ？ @JamesHenry

**Comment by ghiscoding** [comment]
I think what you're looking for the [`publish --canary`](https://github.com/lerna/lerna/tree/main/commands/publish#--canary) option and also possibly the `--preid alpha` option as well

**Reply by whatwg6** [reply]
with --canary command does not generate any changelog

**Reply by ghiscoding** [reply]
well you also need to use [`--conventional-commits`](https://github.com/lerna/lerna/tree/main/commands/version#--conventional-commits), if you are new to Lerna I would suggest you watch some learning videos, for example this one helped me a lot to get started [How to Use Lerna](https://www.youtube.com/watch?v=p6qoJ4apCjA)

**Reply by whatwg6** [reply]
Sorry,I think you don't understand my needs

**Comment by JamesHenry** [comment, marked as answer]
@whatwg6 it's been a while since that work was done, but I have a feeling in the case of v5 I passed an explicit version to `lerna version` myself, i.e. `lerna version 5.0.0-alpha.0 --force-publish`

Hope that helps!

**Comment by whatwg6** [comment]
Close question.
